command/validate/validate_test.go

```diff
@@ -68,6 +125,25 @@ func TestValidateCommand_SucceedOnMinimalConfigDir(t *testing.T) {
 	}
 }
 
+func TestValidateCommand_FailForInvalidJSONConfigFormat(t *testing.T) {
+	t.Parallel()
+	td := testutil.TempDir(t, "consul")
+	defer os.RemoveAll(td)
+
+	fp := filepath.Join(td, "hcl.conf")
+	err := ioutil.WriteFile(fp, []byte(`bind_addr = "10.0.0.1"\ndata_dir = "`+td+`"`), 0644)
+	if err != nil {
+		t.Fatalf("err: %s", err)
```

Comment:
Would like to see us using assert or require in new code - replace with `require.Nil(err) here and `require.Equal(0, exitCode)` below and in other tests.